packages/lib/src/LgIdProps/index.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Base interface used by components to support data-lgid.
+ *
+ * @public
+ */
+export interface LgIdProps {
+  /**
+   * LG test id passed to the component wrapper.
+   */
+  ['data-lgid']?: string;
```

Comment:
A neat thing you could consider if we wanted to validate the ids with TS is using TS template strings:

```typescript
{
  ['data-lgid']?: `lg-${string}`; // enforce the lg-id starts with "lg-"
}
```

Reply:
Though this would get tricky if we're storing the id strings as variables...

Reply:
Looks like this works with variables